packages/scripts/config/webpack.config.js

```diff
@@ -74,6 +75,19 @@ const config = {
 				default: false,
 			},
 		},
+		minimizer: [
+			new TerserPlugin( {
+				cache: true,
+				parallel: true,
+				sourceMap: ! isProduction,
+				terserOptions: {
+					output: {
+						comments: /translators:/i,
```

Comment:
Just found this. Why would we want to keep the translators comments in our production builds? I thought they're only used on our source code?

Reply:
They are needed because the production builds are the ones that are scanned for any translation function calls and made available on translate.wordpress.org.

Reply:
It's really all explained in the ticket this PR addresses: #19006

Reply:
Oh cool! Probably worth a comment here though. Thanks for the info 👍!